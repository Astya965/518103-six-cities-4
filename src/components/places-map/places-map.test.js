import React from "react";
import renderer from "react-test-renderer";

import PlacesMap from "./places-map.jsx";

it(`Render PlacesMap`, () => {
  const tree = renderer.
    create(
        <PlacesMap />
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
