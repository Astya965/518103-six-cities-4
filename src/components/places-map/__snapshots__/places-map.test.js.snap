// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render PlacesMap 1`] = `
<section
  className="property__map map"
/>
`;
